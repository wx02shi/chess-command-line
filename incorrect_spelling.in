game human human
moev e2 e4
move e2 e4
meov e7 e6
move e7 e6
