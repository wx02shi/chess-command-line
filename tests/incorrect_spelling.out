8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 pppp_ppp
6  _ _p_ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh
Final Score:
White: 0
Black: 0
